src/test/java/com/checkmarx/flow/cucumber/integration/end2end/genericendtoend/Repository.java

```diff
@@ -669,39 +661,186 @@ void verifyPRUpdated(String engine) {
             throw new PendingException();
         }
 
+    },
+    GITLAB {
+
+        private static final String GET_PROJECT_URL = "/projects?search=";
+        private static final String GET_HOOKS_URL = "/projects/{id}/hooks";
+        private static final String CREATE_WEBHOOK_URL = "/projects/{id}/hooks?url={webhook}&token={token}";
+        private static final String DELETE_WEBHOOK_URL = "/projects/{id}/hooks/{webhookId}";
+        private static final String COMMIT_URL = "/projects/{id}/repository/commits";
+        private static final String CODE_FILE_PATH = "/encode.frm";
+        private static final String GITLAB_CONFIG_AS_CODE_FILE = "cx.gitlab.configuration";
+        private static final String GITLAB_CONFIG_AS_CODE_PATH = "/cx.gitlab.configuration";
```

Comment:
These 2 constants seem to refer to the same thing. One of them should be deleted.